README.md

```diff
@@ -191,6 +191,23 @@ To access the admin web UI of Rspamd point the browser to
 
     https://<mail host>/rspamd/
 
+## Rspamd bulk Bayesian filter training
+
+To quickly train the Rspamd Bayesian filter plugin, execute the `rspamc`
+wrapper command, which incorporates the Rspamd authentication header
+logic. For example, get `rspamc` help output with:
+
+    runagent -m mail1 podman exec -i dovecot rspamc-wrapper --help
+
+To read SPAM messages from stdin perform a shell redirection, like:
+
+    runagent -m mail1 podman exec -i dovecot rspamc-wrapper learn_spam < spamarchive.mbox
+
```

Comment:
I cannot train rspamd like we did with [NS7](https://wiki.nethserver.org/doku.php?id=userguide:manual_training_of_the_bayes_filter_rspamd). The most hard part is that the spam are not inthe container
```
mkdir spam
cd spam
wget http://untroubled.org/spam/2024-01.7z
7z e 2024-01.7z
# we need to push to the container the spam mail
tar -cvf files.tar *.txt
podman cp files.tar dovecot:/tmp
podman exec dovecot tar -xvf /tmp/files.tar -C /tmp/spam
# now we can make to learn 
podman exec -i dovecot /usr/local/lib/dovecot/sieve-pipe/rspamc-wrapper learn_spam /tmp/spam
# now we can verify
podman exec -i dovecot /usr/local/lib/dovecot/sieve-pipe/rspamc-wrapper stat
```

```
Results for command: stat (0.06 seconds)
Messages scanned: 0
Messages learned: 475
Connections count: 0
Control connections count: 518
Pools allocated: 571
Pools freed: 536
Bytes allocated: 26470776
Memory chunks allocated: 293
Shared chunks allocated: 3
Chunks freed: 0
Oversized chunks: 333
Fuzzy hashes in storage "rspamd.com": 10610683149
Fuzzy hashes stored: 10610683149
Statfile: BAYES_SPAM type: redis; length: 0; free blocks: 0; total blocks: 0; free: 0.00%; learned: 0; users: 0; languages: 0
Statfile: BAYES_HAM type: redis; length: 0; free blocks: 0; total blocks: 0; free: 0.00%; learned: 0; users: 0; languages: 0
Total learns: 0
```